pkg/ruler/ruler.go

```diff
@@ -161,7 +159,8 @@ type Config struct {
 	EnableQueryStats      bool `yaml:"query_stats_enabled"`
 	DisableRuleGroupLabel bool `yaml:"disable_rule_group_label"`
 
-	EnableHAEvaluation bool `yaml:"enable_ha_evaluation"`
+	EnableHAEvaluation       bool          `yaml:"enable_ha_evaluation"`
+	EvalHAHealthCheckTimeout time.Duration `yaml:"eval_ha_health_check_timeout"`
```

Comment:
I don't like the flag name... But I don't have a good suggestion.

Reply:
Changed it to `LivenessCheckTimeout`